styles/themes/components/button.scss

```diff
@@ -42,4 +42,32 @@
 .button-ghost:hover{
     border-color:var(--rad-ui-color-accent-950) ;
     background-color: var(--rad-ui-color-accent-300);
+}
+
+/*small */
+[data-size="small"]{
```

Comment:
perfect, we can move all these size classes inside the `rad-ui-button` CSS definition

otherwise these styles will end up applying to all HTML elements and not just buttons

Reply:
Can you please give some example

Reply:
currently we do it like this

```css
.rad-ui-button{
}
[data-size="small"]{
}
```

We need to do this

```css
.rad-ui-button{
    [data-size="small"]{
     }
}

```

Reply:
There is some styling applied on the rad ui button you want to replace it with data-size attributes or you wanna keep it.

Reply:
no need for replacing - those styles are important and shouldn't be replaced, just place these chunks of code below the existing styles, it should work as expected.

it's SCSS nesting syntax if you have never worked on it before